org.eclipse.jdt.core.tests.compiler/src/org/eclipse/jdt/core/tests/compiler/regression/TryStatementTest.java

```diff
@@ -1499,14 +1499,15 @@ public void test033() throws Exception {
 }
 //https://bugs.eclipse.org/bugs/show_bug.cgi?id=124853
 public void test034() throws Exception {
+	String builder = this.complianceLevel >= ClassFileConstants.JDK1_5 ? "StringBuilder" : "StringBuffer";
 	this.runConformTest(
 			new String[] {
 				"X.java",
 				"public class X {\n" +
 				"	private static int scenario(){\n" +
 				"		try {\n" +
 				"			int i = 1;\n" +
-				"			System.out.print(\"[i: \" + i+\"]\");\n" +
+				"			System.out.print(new " + builder + "(\"[i: \").append(i).append(\"]\").toString());\n" +
```

Comment:
Surprised by this change - what brought this about ? Is this a way of short circuiting the new indy based concat ?

Reply:
Yes, I didn't want to change all the tests' output that involved a String concat. I kept few to validate the new generated code, though.